### PR TITLE
docs: pause process with multisig

### PR DESCRIPTION
**Pull request by JuArce**
# Pause process with multisig wallet

## Description

This PR adds docs for the pause process using a multisig for the AlignedLayerServiceManager or the BatcherPaymentService

## Type of change

- [x] Documentation

### CODE REVIEW

**Comment by PatStiles** [comment]
We should add a section on verifying via cast or another tool that the contract is `paused`.

File: docs/0_internal/4_a_pause_contracts.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Pause Contracts
+This doc contains a guide on how to use the Pausable functionality of Aligned.
+
+To run the make targets specified in this guide, you must first have the relevant following env vars:
```
Comment (MarcosNicolau) [inline docs/0_internal/4_a_pause_contracts.md]: I would the path to the `.env`
```suggestion
To run the make targets specified in this guide, you must first have the relevant following env vars under `contracts/scripts/.env`:
```

**Review by MarcosNicolau — APPROVED** [review]

File: docs/0_internal/4_a_pause_contracts.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+5. receive
+
+## BatcherPaymentsService
+
+BatcherPayments is also pausable, but without so much detail. You can either pause or unpause the contract running the following:
+
+- Get current paused state:
+```
+make get_paused_state_batcher_payments_service
+```
```
Comment (avilagaston9) [inline docs/0_internal/4_a_pause_contracts.md:62]: Get the following error when running this:
```bash
/bin/bash: contracts/scripts/get_paused_state_batcher_payments_service.sh: No such file or directory
```

File: docs/0_internal/4_a_pause_contracts.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+Aligned Service Manager is granulary pausable, which means you can pause the whole contract, or only specific functions. For this,
+Aligned uses the Pauser Registry contract provided by Eigenlayer. This contract stores the role of different accounts, so
+you can have X pausers and Y unpausers.
+
+To interact with it you can:
+
+- Get current paused state:
+```
+make get_paused_state_aligned_service_manager
+```
```
Comment (avilagaston9) [inline docs/0_internal/4_a_pause_contracts.md:24]: `source contracts/scripts/.env` is needed before running all targets in this doc

**Review by PatStiles — APPROVED** [review]

File: docs/0_internal/4_a_pause_contracts.md (hunk 2 above)
Comment (uri-99) [reply docs/0_internal/4_a_pause_contracts.md:62]: +1, working on the fix

File: docs/0_internal/4_a_pause_contracts.md (hunk 3 above)
Comment (uri-99) [reply docs/0_internal/4_a_pause_contracts.md:24]: wip fix

**Review by uri-99 — CHANGES_REQUESTED** [review]
req changes until i do some fixes

File: docs/0_internal/4_a_pause_contracts.md (hunk 3 above)
Comment (uri-99) [reply docs/0_internal/4_a_pause_contracts.md:24]: fixed on https://github.com/yetanotherco/aligned_layer/pull/1501

File: docs/0_internal/4_a_pause_contracts.md (hunk 2 above)
Comment (uri-99) [reply docs/0_internal/4_a_pause_contracts.md:62]: fixed on https://github.com/yetanotherco/aligned_layer/pull/1501

File: docs/0_internal/4_b_1_propose_pause.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
+# Propose the Transaction for Pause using Multisig
+
+If you want to pause the contracts, you can propose the pause transaction using the multisig wallet.
+
+## Prerequisites
+
+- You need to have deployed the contracts following the [Deploy Contracts Guide](./2_deploy_contracts.md).
```
Comment (uri-99) [inline docs/0_internal/4_b_1_propose_pause.md:7]: broken link, wip fix

File: docs/0_internal/4_b_2_approve_pause.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+Once the transaction is proposed, the multisig owners must approve the transaction.
+
+## Approve the Pause for AlignedLayerServiceManager
+
+1. Go to [Safe](https://app.safe.global/home) and connect your wallet.
+
+2. Go to the `Transactions` tab and find the transaction that was proposed.
+
+3. Get the ```pauseAll()``` signature by running:
```
Comment (avilagaston9) [inline docs/0_internal/4_b_2_approve_pause.md:11]:
```suggestion
3. Get the ```pauseAll()``` signature by running in your terminal:
```

File: docs/0_internal/4_b_1_propose_pause.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+## Propose transaction for Pause AlignedLayerServiceManager
+
+To propose the pause transaction you can follow the steps below:
+
+1. Go to [Safe](https://app.safe.global/home)
```
Comment (avilagaston9) [inline docs/0_internal/4_b_1_propose_pause.md:13]:
```suggestion
1. Go to [Safe](https://app.safe.global/home) and connect your wallet.
```

**Review by uri-99 — APPROVED** [review]

**Review by avilagaston9 — APPROVED** [review]
Works in my machine. Left some minor comments

File: docs/0_internal/4_b_pause_contracts_with_multisig.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+1. Propose the transaction with the multisig following the [Propose Pause Guide](./4_b_1_propose_pause.md).
+
+2. After the pause is proposed, multisig participants can approve the pause following the [Approve Pause Guide](./4_b_2_approve_pause.md).
+
+## Steps for Unpausing
+
+1. Propose the transaction with the multisig following the [Propose Unpause Guide](./4_b_3_propose_unpause.md).
+
+2. After the unpause is proposed, multisig participants can approve the unpause following the [Approve Unpause Guide](./4_b_4_approve_unpause.md).
```
Comment (uri-99) [inline docs/0_internal/4_b_pause_contracts_with_multisig.md:47]: these links are added in [this pr](https://github.com/yetanotherco/aligned_layer/pull/1497/files)

**Pull request merged**